src/analyze.ts

```diff
@@ -617,12 +629,20 @@ export async function runQueries(
     queryFlags.push("--extension-packs=codeql-action/pr-diff-range");
     incrementalMode.push("diff-informed");
   }
+
   if (
     config.augmentationProperties.overlayDatabaseMode ===
     OverlayDatabaseMode.Overlay
   ) {
+    statusReport.analysis_is_overlay = true;
     incrementalMode.push("overlay");
+  } else if (
+    config.augmentationProperties.overlayDatabaseMode ===
+    OverlayDatabaseMode.OverlayBase
+  ) {
+    statusReport.analysis_builds_overlay_base_database = true;
```

Comment:
Minor: perhaps this would be slightly nicer as, similar to what we already have for `analysis_is_diff_informed`: 
```suggestion
  } 
  
  statusReport.analysis_builds_overlay_base_database =
    config.augmentationProperties.overlayDatabaseMode ===
    OverlayDatabaseMode.OverlayBase;
```